draft-add-requirements.md

```diff
@@ -55,66 +92,119 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD",
+can provide several benefits that are not possible if only unencrypted DNS is used:
+
+- Prevent other devices on the network from observing client DNS messages
+- Verify that answers come from the selected DNS resolver
+- Authenticate that the DNS resolver is the one provisioned by the network
+
+Often, network-provisioned resolvers are forwarders running on a local router. The discovered
+encrypted resolvers in these cases may either be local fowarders themselves, or an associated
+resolver that is in the network (thus bypassing the router's DNS forwarder).
 
 | Requirement | Description |
 | R2.1 | Example requirement |
 
 ### Unencrypted forwarder
 
-If the resolver announced by the network is a classic unencrypted forwarder, it is frequently the case that such forwarders are difficult to upgrade
-to support encrypted operation. In such cases it is useful for the resolver provider to be able to declare which encrypted resolvers they provide,
-and for the client to be able to discover them. If the client wishes to, it can then use one of those resolvers and bypass the local forwarder.
+If the resolver announced by the network is a classic unencrypted forwarder, it is frequently the case that such
+forwarders are difficult to upgrade to support encrypted operation. In such cases it is useful for the resolver
+provider to be able to declare which encrypted resolvers they provide, and for the client to be able to discover
+them. If the client wishes to, it can then use one of those resolvers and bypass the local forwarder.
 
 | Requirement | Description |
 | R3.1 | Example requirement |
 
 ### Encrypted forwarder
 
-If a subset of local resolvers supports encrypted DNS, the client may not initially be aware that its local resolver supports it. Discovering this
-may require communication with the local resolver, or an upstream resolver, over an unencrypted transport. Once discovered, the local encrypted forwarder
-may be selected by the client, gaining the benefits of encryption while retaining the benefits of a local caching forwarder with knowledge of the local
-topology.
+If a subset of local resolvers supports encrypted DNS, the client may not initially be aware that its local resolver
+supports it. Discovering this may require communication with the local resolver, or an upstream resolver, over an
+unencrypted transport. Once discovered, the local encrypted forwarder may be selected by the client, gaining the
+benefits of encryption while retaining the benefits of a local caching forwarder with knowledge of the local topology.
 
 | Requirement | Description |
 | R4.1 | Example requirement |
 
-## Discover resolvers associated with a manually-configured resolver
+## Client-selected resolvers {#client-selected}
+
+Client devices often allow a user or administrator to select a specific DNS resolver to use
+on certain networks, or on all networks. Historically, this selection was specified only with an
+IP address.
```

Comment:
Why would the client allow a administrator to select a specific DNS resolver to use in all network unless it is a managed device by the Enterprise network ?

Reply:
Are you saying the user would object to this, or the client device's operating system? Because in general any admin can set /etc/resolv.conf, or the equivalent.

Reply:
I meant users may not want the Enterprise IT administrator to select a specific DNS resolver to be used in all networks for their BYOD.  We should probably update the text to say:
Client devices often allow the device administrator to select a specific DNS resolver to use on certain networks, or on all networks.

Reply:
Done.